README.md

```diff
@@ -1,2 +1,66 @@
-# reggaemedia
-the reggaemedia website
+# Getting started
+1. Before starting development server you need to define environment variables.
+2. To be able to deploy project stack, your PATH should include executable aws package with configured IAM account.
+3. Preferred package manager is `yarn` version 2.
+
+## .ENV file
+Environment variables file must include vars written below to be able to deploy AWS stack. Vars below is stale and should be define once for whole project.
+
+```
+COGNITO_CALLBACK_URL=http://localhost:3000/api/auth/callback/cognito
+ADMIN_EMAIL
+CDK_DEFAULT_ACCOUNT= default account that is being defined in aws configuration
+CDK_DEFAULT_REGION= preferred region
+AMAZON_ACCESS_KEY= IAM credentials
+AMAZON_SECRET_KEY= IAM credentials
+AMAZON_REGION= preferred region
+```
+
+After defining variables above you should run command `yarn stack:deploy`. That command will generate `cdk-outputs.json` that is being gitignored, output file contain another variables that must be defined in the same .env file as above. 
+
+### WARNING:
+After running `yarn stack:deploy`, you always have to update your .env file, since stack is being regenerated.
+
+```
+{
+  "reggae-media-stack": {
+    "bucket",
+    "settings",
+    "userPoolClientId",
+    "userPoolClientSecret",
+    "articles",
+    "userPoolId",
+  }
+}
+```
+
+Environments below should contain data that is being generated by aws-cdk-lib in `cdk-outputs.json`.
+
+```
+BUCKET_NAME="bucket"
+TABLE_ARTICLES="articles"
+COGNITO_CLIENT_SECRET="userPoolClientSecret"
+COGNITO_CLIENT_ID="userPoolClientId"
+COGNITO_USER_POOL_ID="userPoolId"
+TABLE_SETTINGS="settings"
+```
+
+## Development process
+After deploying your stack and defining all the variables run command `yarn dev` to start development process.
```

Comment:
your stack -> the stack.

Reply:
actually, we can rewrite this part also, like

When the preparation is completed the `yarn dev` command will start the development server.